src/Features/Core/Portable/CodeRefactorings/MoveType/AbstractMoveTypeService.cs

```diff
@@ -58,15 +58,20 @@ public override async Task<Solution> GetModifiedSolutionAsync(Document document,
                 return document.Project.Solution;
             }
 
-            var suggestedFileNames = GetSuggestedFileNames(
-                state.TypeNode,
-                IsNestedType(state.TypeNode),
-                state.TypeName,
-                state.SemanticDocument.Document.Name,
-                state.SemanticDocument.SemanticModel,
-                cancellationToken);
-
-            var editor = Editor.GetEditor(operationKind, (TService)this, state, suggestedFileNames.FirstOrDefault(), cancellationToken);
+            var suggestedFileName = operationKind switch
+            {
+                MoveTypeOperationKind.RenameType => Path.GetFileNameWithoutExtension(document.Name),
```

Comment:
slightly confsued . if we're renaming the type... the suggested file name is the name the document already is?  can you doc what is going on here?